packages/next-on-pages/env.d.ts

```diff
@@ -5,6 +5,7 @@ declare global {
 			npm_config_user_agent?: string;
 			CF_PAGES?: string;
 			SHELL?: string;
+			KV_SUSPENSE_CACHE?: KVNamespace;
```

Comment:
```suggestion
			__NEXT_ON_PAGES__KV_SUSPENSE_CACHE?: KVNamespace;
```
(
or
```suggestion
			NEXT_ON_PAGES_KV_SUSPENSE_CACHE?: KVNamespace;
```
if the suggestion above is too much
)

I think it would be nice/beneficial to make it very obvious that this is a next-on-pages thing, so that people don't end up risking to confuse these with bindings they actually directly use, thinking that they can be Cloudflare internal ones (like the sites ones) etc...

Reply:
what about `NOP_KV_SUSPENSE_CACHE`?

Reply:
😢 

since it's just a binding that you can set and forget I don't see an issue in a longer (clearer/more recognizable) name

I guess you find `__NEXT_ON_PAGES__KV_SUSPENSE_CACHE` too too long? 😢 

(PS: if we do add a next-on-pages prefix, then we could actually simplify the latter part of the name, for example by dropping `SUSPENSE`? 🤔)

Reply:
i guess `__NEXT_ON_PAGES__KV_CACHE` works, just kinda long 😓

Reply:
or maybe `NEXT_ON_PAGES_KV_CACHE` without the extra underscores, would that be okay?

Reply:
I was thinking of the extra underscores as a separation between `NEXT_ON_PAGES` and the rest so that it could be a nice prefix indicator for any potential other bindings we might want to add
(e.g. `__NEXT_ON_PAGES__KV_CACHE`, `__NEXT_ON_PAGES__xxx`, ...)

I think it'd be nice, but if you really dislike it I can concede and not having them 🙂

Reply:
but people might make a typo is what i was thinking

Reply:
I would assume most people not typing it but copy-pasting it from the docs anyways 🤔 🤷 

But yeah up to you, I'm fine with not having the double slashes if you're really concerned about that 🙂

Reply:
I like `__NEXT_ON_PAGES__KV_SUSPENSE_CACHE` FWIW.